<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>yesNoCheckDialog</class>
 <widget class="QDialog" name="yesNoCheckDialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>128</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Project loaded</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QLabel" name="question">
     <property name="text">
      <string>Do you want to regenerate and overwrite sif file?</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>    Don't ask me again</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="yesButton">
       <property name="text">
        <string>Yes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="noButton">
       <property name="text">
        <string>No</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../ElmerGUI.qrc"/>
 </resources>
 <connections/>
</ui>
